CppImport/src/Comment.cpp

```diff
@@ -78,8 +78,41 @@ void Comment::removeFromItemList()
 QString Comment::text()
 {
 	auto lines = text_.split('\n');
-	for (auto i = 0; i < lines.length(); i++)
-		lines[i] = lines[i].trimmed();
+	if (!lines.first().startsWith("/*"))
+	{
+		QRegularExpression singleLineRegex{"^\\s*//\\s*([^\\s].*)$", QRegularExpression::DotMatchesEverythingOption};
+		for (auto i = 0; i < lines.size(); i++)
+		{
+			auto match = singleLineRegex.match(lines[i]);
+			if (match.hasMatch()) lines[i] = match.captured(1);
+		}
+	}
+	else
+	{
+		QRegularExpression firstLineRegex{"^/\\*+<?\\s?(.*)$", QRegularExpression::DotMatchesEverythingOption};
+		auto match = firstLineRegex.match(lines.first());
+		if (match.hasMatch()) lines[0] = match.captured(1);
+
+		QRegularExpression middleLineRegex{"^\\s*\\*\\s(.*)$", QRegularExpression::DotMatchesEverythingOption};
+		for (auto i = 1; i < lines.size(); i++)
+		{
+			match = middleLineRegex.match(lines[i]);
+			if (match.hasMatch()) lines[i] = match.captured(1);
+		}
+
+		QRegularExpression lastLineRegex{"^(.*)\\*/\\s*$", QRegularExpression::DotMatchesEverythingOption};
```

Comment:
I am not sure this handles the `**********/` case. We need to have `\\*+` but for this to work it needs to be preceeded by something that is not a `*`. Could you adjust the regex accordingly?